### ISSUE TITLE
Acquia CLI segfaults on Mac M1

### ISSUE THREAD

**Issue by froboy** [docksal/docksal#1616]
**Description**

Acquia CLI fails on M1 Macs. See https://github.com/drud/ddev/issues/3116.

**Steps to reproduce the issue:**

1. `fin acli auth:login`
2. Enter your key and secret

**Describe the results you received:**

`Segmentation fault`

**Describe the results you expected:**

Not a seg fault. 

**Output of `fin config`:**

<details>
  <summary>fin config output</summary>

```
---------------------
COMPOSE_PROJECT_NAME_SAFE: ymca-nyc
COMPOSE_FILE:
/Users/avi/.docksal/stacks/volumes-none.yml
/Users/avi/.docksal/stacks/stack-default.yml
/Users/avi/Projects/ymca-nyc/.docksal/docksal.yml
ENV_FILE:
/Users/avi/Projects/ymca-nyc/.docksal/docksal.env
/Users/avi/Projects/ymca-nyc/.docksal/docksal-local.env

PROJECT_ROOT: /Users/avi/Projects/ymca-nyc
DOCROOT: docroot
VIRTUAL_HOST: ymca-nyc.docksal
VIRTUAL_HOST_ALIASES: *.ymca-nyc.docksal
IP: 192.168.64.100
MYSQL: 192.168.64.100:62197

Docker Compose configuration
---------------------
services:
  adminer:
    environment:
      ADMINER_DEFAULT_SERVER: db
      ADMINER_DESIGN: price
      ADMINER_PLUGINS: tables-filter tinymce
    hostname: adminer
    image: adminer
    labels:
      io.docksal.virtual-host: adminer-ymca-nyc.docksal
  cli:
    dns:
    - 192.168.64.100
    - 8.8.8.8
    - 192.168.64.100
    - 8.8.8.8
    - 192.168.64.100
    - 8.8.8.8
    environment:
      BLACKFIRE_CLIENT_ID: null
      BLACKFIRE_CLIENT_TOKEN: null
      COMPOSER_ALLOW_XDEBUG: '0'
      COMPOSER_DEFAULT_VERSION: null
      COMPOSER_DISABLE_XDEBUG_WARN: '0'
      DOCROOT: docroot
      DRUSH_ALLOW_XDEBUG: '0'
      DRUSH_OPTIONS_URI: ymca-nyc.docksal
      GIT_USER_EMAIL: ...
      GIT_USER_NAME: Avi Schwab
      HOST_GID: '20'
      HOST_UID: '501'
      MYSQL_DATABASE: default
      MYSQL_HOST: db
      MYSQL_PASSWORD: user
      MYSQL_ROOT_PASSWORD: root
      MYSQL_USER: user
      PHP_IDE_CONFIG: null
      SECRET_ACAPI_EMAIL: <redacted>
      SECRET_ACAPI_KEY: <redacted>
      SECRET_ACQUIACLI_KEY: null
      SECRET_ACQUIACLI_SECRET: null
      SECRET_PLATFORMSH_CLI_TOKEN: null
      SECRET_SSH_PRIVATE_KEY: null
      SECRET_TERMINUS_TOKEN: null
      SSH_AUTH_SOCK: /.ssh-agent/proxy-socket
      VIRTUAL_HOST: ymca-nyc.docksal
      XDEBUG_CONFIG: remote_connect_back=0 remote_host=192.168.64.1
      XDEBUG_ENABLED: '0'
    healthcheck:
      interval: 10s
    hostname: cli
    image: docksal/cli:php7.4-3.0
    labels:
      io.docksal.shell: bash
      io.docksal.user: docker
    logging:
      options:
        max-file: '10'
        max-size: 1m
    volumes:
    - docksal_ssh_agent:/.ssh-agent:ro
    - /home/docker
    - /tmp/.docksal/ymca-nyc:/tmp/.docksal/ymca-nyc:ro
    - project_root:/var/www:rw,nocopy
  db:
    dns:
    - 192.168.64.100
    - 8.8.8.8
    - 192.168.64.100
    - 8.8.8.8
    - 192.168.64.100
    - 8.8.8.8
    environment:
      MYSQL_ALLOW_EMPTY_PASSWORD: null
      MYSQL_DATABASE: default
      MYSQL_INITDB_SKIP_TZINFO: null
      MYSQL_ONETIME_PASSWORD: null
      MYSQL_PASSWORD: user
      MYSQL_RANDOM_ROOT_PASSWORD: null
      MYSQL_ROOT_PASSWORD: root
      MYSQL_USER: user
    healthcheck:
      interval: 10s
    hostname: db
    image: docksal/mysql:5.7-1.5
    logging:
      options:
        max-file: '10'
        max-size: 1m
    ports:
    - target: 3306
    volumes:
    - db_data:/var/lib/mysql:rw
    - project_root:/var/www:ro,nocopy
  pma:
    environment:
      PMA_HOST: db
      PMA_PASSWORD: root
      PMA_USER: root
    hostname: pma
    image: phpmyadmin/phpmyadmin
    labels:
      io.docksal.virtual-host: pma.ymca-nyc.docksal
  solr:
    dns:
    - 192.168.64.100
    - 8.8.8.8
    healthcheck:
      interval: 10s
    hostname: solr
    image: docksal/solr:1.0-solr4
    labels:
      io.docksal.cert-name: none
      io.docksal.virtual-host: solr.ymca-nyc.docksal,solr.ymca-nyc.docksal.*
      io.docksal.virtual-port: '8983'
    logging:
      options:
        max-file: '10'
        max-size: 1m
    volumes:
    - project_root:/var/www:ro,nocopy,cached
  web:
    depends_on:
      cli:
        condition: service_started
    dns:
    - 192.168.64.100
    - 8.8.8.8
    - 192.168.64.100
    - 8.8.8.8
    - 192.168.64.100
    - 8.8.8.8
    environment:
      APACHE_BASIC_AUTH_PASS: null
      APACHE_BASIC_AUTH_USER: null
      APACHE_DOCUMENTROOT: /var/www/docroot
      APACHE_FCGI_HOST_PORT: cli:9000
    healthcheck:
      interval: 10s
    hostname: web
    image: docksal/apache:2.4
    labels:
      io.docksal.cert-name: none
      io.docksal.permanent: "false"
      io.docksal.project-root: /Users/avi/Projects/ymca-nyc
      io.docksal.virtual-host: ymca-nyc.docksal,*.ymca-nyc.docksal
    logging:
      options:
        max-file: '10'
        max-size: 1m
    volumes:
    - project_root:/var/www:ro,nocopy
version: '2.1'
volumes:
  cli_home: {}
  db_data: {}
  docksal_ssh_agent:
    external: true
    name: docksal_ssh_agent
  project_root: {}

---------------------
```
</details>

<br>

**Output of `fin sysinfo`:**

<details>
  <summary>fin sysinfo output</summary>

  ```

███  FIN
fin version: 1.107.1

███  OS
Darwin macOS 12.1
Darwin Avis-MacBook-Pro.local 21.2.0 Darwin Kernel Version 21.2.0: Sun Nov 28 20:28:41 PST 2021; root:xnu-8019.61.5~1/RELEASE_ARM64_T6000 arm64

███  ENVIRONMENT
MODE : Docker Desktop
DOCKER_HOST : 

███  NFS
DOCKSAL_NFS_PATH : /Users

nfsd service is enabled
nfsd is running (pid 478, 8 threads)

NFS EXPORTS
----------

# <ds-nfs docksal
/Users 127.0.0.1 192.168.64.1 -alldirs -maproot=0:0
# ds-nfs>
----------

Exports list on localhost:
/Users                              127.0.0.1 192.168.64.1

███  DOCKER
EXPECTED CLIENT VERSION: 20.10.7
EXPECTED SERVER VERSION: 20.10.7

Client:
 Version:           20.10.7
 API version:       1.41
 Go version:        go1.16.4
 Git commit:        f0df350
 Built:             Wed Jun  2 11:56:22 2021
 OS/Arch:           darwin/amd64 (rosetta)
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.10
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.16.9
  Git commit:       e2f740d
  Built:            Mon Oct 25 07:41:10 2021
  OS/Arch:          linux/arm64
  Experimental:     false
 containerd:
  Version:          1.4.11
  GitCommit:        5b46e404f6b9f661a205e28d59c982d3634148f8
 runc:
  Version:          1.0.2
  GitCommit:        v1.0.2-0-g52b36a2
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0

███  DOCKER COMPOSE
EXPECTED VERSION: 1.29.2
docker-compose version 1.29.2, build 5becea4c
docker-py version: 5.0.0
CPython version: 3.9.0
OpenSSL version: OpenSSL 1.1.1h  22 Sep 2020

███  DOCKSAL: PROJECTS
project                 STATUS                   virtual host                                                project root
ymca-nyc                Up 3 minutes (healthy)   ymca-nyc.docksal,*.ymca-nyc.docksal                         /Users/avi/Projects/ymca-nyc


███  DOCKSAL: VIRTUAL HOSTS
[0;32;49m*.openy-dev.docksal[0m
[0;32;49m*.ymca-nyc.docksal[0m
...

███  DOCKSAL: NETWORKING

DOCKSAL_IP: 192.168.64.100
DOCKSAL_HOST_IP: 192.168.64.1
DOCKSAL_VHOST_PROXY_IP: 0.0.0.0
DOCKSAL_DNS_IP: 0.0.0.0
DOCKSAL_DNS_DISABLED: 0
DOCKSAL_NO_DNS_RESOLVER: 0
DOCKSAL_DNS_UPSTREAM: 
DOCKSAL_DNS_DOMAIN: docksal

███  DOCKSAL: CONNECTIVITY

Host to 192.168.64.100: 	PASS
Container to 192.168.64.100: 	PASS
Container to 192.168.64.1: 	PASS

Checking connectivity to http://dns-test.docksal...
Host: PASS
Containers: PASS

███  DOCKER: RUNNING CONTAINERS
CONTAINER ID   IMAGE                      COMMAND                  CREATED          STATUS                   PORTS                                      NAMES
3ff2b8adc328   docksal/apache:2.4         "httpd-foreground"       3 minutes ago    Up 3 minutes (healthy)   80/tcp, 443/tcp                            ymca-nyc_web_1
a7505a98cd28   docksal/mysql:5.7-1.5      "docker-entrypoint.s…"   3 minutes ago    Up 3 minutes (healthy)   33060/tcp, 0.0.0.0:53940->3306/tcp         ymca-nyc_db_1
3840ef523c0d   docksal/cli:php7.4-3.0     "/opt/startup.sh sup…"   3 minutes ago    Up 3 minutes (healthy)   22/tcp, 3000/tcp, 9000/tcp                 ymca-nyc_cli_1
...

███  DOCKER: NETWORKS
NETWORK ID     NAME                            DRIVER    SCOPE
88eddfed8bfa   _default                        bridge    local
8be89f8e6e52   bridge                          bridge    local
7f2356cdb2da   host                            host      local
88aaba5efc6f   none                            null      local
...
c21e0f0fddc7   ymca-nyc_default                bridge    local
...

███  DOCKER DESKTOP
EXPECTED VERSION: 3.4.0
DETECTED VERSION: 4.2.0

███  HDD Usage
Filesystem       Size   Used  Avail Capacity iused      ifree %iused  Mounted on
/dev/disk3s1s1  926Gi   15Gi  742Gi     2%  577263 4293788033    0%   /
devfs           200Ki  200Ki    0Bi   100%     692          0  100%   /dev
/dev/disk3s6    926Gi   20Ki  742Gi     1%       0 7784777240    0%   /System/Volumes/VM
/dev/disk3s2    926Gi  336Mi  742Gi     1%    1184 7784777240    0%   /System/Volumes/Preboot
/dev/disk3s4    926Gi  2.5Mi  742Gi     1%      43 7784777240    0%   /System/Volumes/Update
/dev/disk1s2    500Mi  6.0Mi  480Mi     2%       1    4914000    0%   /System/Volumes/xarts
/dev/disk1s1    500Mi  7.3Mi  480Mi     2%      31    4914000    0%   /System/Volumes/iSCPreboot
/dev/disk1s3    500Mi  2.0Mi  480Mi     1%      51    4914000    0%   /System/Volumes/Hardware
/dev/disk3s5    926Gi  168Gi  742Gi    19% 1503702 7784777240    0%   /System/Volumes/Data
map auto_home     0Bi    0Bi    0Bi   100%       0          0  100%   /System/Volumes/Data/home

  ```

</details>

**Comment by froboy** [comment]
Related to https://github.com/docksal/docksal/issues/1421

**Comment by lmakarov** [comment]
This should be addressed by https://github.com/docksal/service-cli/issues/265 and is available in the most recent docksal/cli [v3.1.0](https://github.com/docksal/service-cli/releases/tag/v3.1.0) images:

```
docksal/cli:php7.4-3.1
docksal/cli:php8.0-3.1
docksal/cli:php8.1-3.1
```

The upcoming Docksal v1.17 release will use these images.
